lib/bootloader_pvm.pm

```diff
@@ -178,14 +178,16 @@ sub prepare_pvm_installation {
     if (is_agama) {
         record_info("Installing", "Please check the expected product is being installed");
         assert_screen('agama-installer-live-root', 400);
-        return;
     }
-
-    assert_screen("run-yast-ssh", 300);
+    else {
+        assert_screen("run-yast-ssh", 300);
+    }
 
     if (!is_upgrade && !get_var('KEEP_DISKS')) {
```

Comment:
```suggestion
    if (!is_upgrade && !get_var('KEEP_DISKS') !get_var(AGAMA_AUTO)) {
```

Reply:
Yes, it is not needed to format disk for unattended test. Updated, thanks.